guides/satellite-sync.sh

```diff
@@ -3,14 +3,49 @@
 # ##################################################
 # Created on: Fri, 23 Oct 2019
 # Created by: Andrew Dahms
-# Hacked repeately by Melanie Corr to accomodate transitions towards Upstream-Downstream Satellite
+# Hacked repeately by Melanie Corr to accommodate transitions towards Upstream-Downstream Satellite
+# To use the script to synchronize changes from the upstream foreman-documentation GitHub repository to the downstream docs-Red_Hat_Satellite_6 repository, complete the following steps:"
+# 1. Create a directory to store repositories for synchronization. You can name the directory whatever you want, for example SyncCenter:"
+# $ mkdir ~/Documents/SyncCenter"
+#
+# 2. Copy this script to the SyncCenter directory."
+#
+# 3. Change to the SyncCenter directory."
+#
+# 4. In the upstream repository, ensure that the commits that you want to synchronized are cherry-picked to the related branch, for example, to the SATELLITE-6.8-beta branch. In the upstream repository, all related to downstream branches are in the format 'SATELLITE-x.y'"
```

Comment:
In the upstream repository, ensure that the commits you want to synchronize are first cherry-picked to the branch that corresponds to the downstream branch, for example, to the SATELLITE-6.8-beta branch. In the upstream repository, branches that correspond to the downstream branches are in the format 'SATELLITE-x.y'"

Reply:
Done.